intelmq/tests/bots/experts/aggregate/testdata/events.txt

```diff
@@ -0,0 +1,25 @@
+{"__type": "Event", "source.ip": "93.184.216.34", "destination.ip": "192.0.43.8", "time.observation": "2015-01-01T00:00:01+00:00", "classification.type": "ddos", "classification.identifier": "ddos" }
```

Comment:
`93.184.216.34` is not a reserved address, please use another one (the rfc1918 expert has a compact list of neutral network ranges)

Reply:
`93.184.216.34` has been extracted from the rfc1918 tests.

Reply:
https://github.com/certtools/intelmq/blob/2190345b7c0699cbcb18d36132d581c306c9ec6f/intelmq/bots/experts/rfc1918/expert.py#L33-L37 doesn't list it. The address is used in the test https://github.com/certtools/intelmq/blob/2190345b7c0699cbcb18d36132d581c306c9ec6f/intelmq/tests/bots/experts/rfc1918/test_expert.py#L16 because it's *not* a reserved address.

Reply:
I think this discussion is still open